### PR TITLE
Add ORO token address to worldchain

### PR DESCRIPTION
**Pull request by 0xdatawolf001**
### Update!
Please build spells in the proper [subproject](../dbt_subprojects/) directory. For more information, please see the main [readme](../README.md), which also links to a GH discussion with the option to ask questions.

### Contribution type
Please check the type of contribution this pull request is for:

- [ ] New spell(s)
- [X ] Adding to existing spell lineage
- [ ] Bug fix

**Note:** You can safely discard any section below which doesn't apply based on selection above

---

### For new spell(s)
If you are building new spell(s), please provide the following information:

- **Spell name(s):** *schema.table_name*
- **Description:** [Detailed description of the new spell(s) and their purpose]
- **Who are the new spell(s) for?** [Internal team or general community]
- **How will the new spell(s) be used downstream?** [Description of downstream usage]
- **Implementation details:** [Information on how the spell(s) are implemented]
- **Test instructions:** [How to test the new spell(s)]
- **Related issue(s):** [Link to related issues, if any]

---

### For adding to existing spell lineage
If you are adding to an existing spell lineage, please provide the following information:

- **Description:** [Description of the changes made]
- Addition of a new token

---

### For bug fixes
If you are fixing a bug, please provide the following information:

- **Description:** [Description of the bug fix]
- **Steps to reproduce:** [How to reproduce the bug]
- **Implementation details:** [Information on how the bug was fixed]
- **Test instructions:** [How to test the fix]
- **Related issue(s):** [Link to related issues, if any]

---

### Additional information
Please provide any additional information that might help us review your pull request:

- [Any additional information]

---

Thank you for your contribution!

### CODE REVIEW

**Comment by 0xdatawolf001** [comment]
This is my first PR to spellbook and would probably have botched something. Let me know if something is blocking

**Comment by 0xdatawolf001** [comment]
I have read the CLA Document and I hereby sign the CLA

File: dbt_subprojects/tokens/models/tokens/worldchain/tokens_worldchain_erc20.sql [hunk 1]
```diff
@@ -17,4 +17,5 @@ FROM (VALUES
     , (0x03C7054BCB39f7b2e5B2c7AcB37583e32D70Cfa3, 'WBTC', 8)
     , (0x4200000000000000000000000000000000000006, 'WETH', 18)
     , (0x859DBE24b90C9f2f7742083d3cf59cA41f55Be5d, 'sDAI', 18)
+    , (0xcd1E32B86953D79a6AC58e813D2EA7a1790cAb63, 'ORO', 18)
```
Comment (Hosuke) [inline dbt_subprojects/tokens/models/tokens/worldchain/tokens_worldchain_erc20.sql:20]: https://worldchain-mainnet.explorer.alchemy.com/token/0xcd1E32B86953D79a6AC58e813D2EA7a1790cAb63

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @0xdatawolf001

**Comment by Hosuke** [comment]
> This is my first PR to spellbook and would probably have botched something. Let me know if something is blocking

Welcome to spellbook!

**Comment by jeff-dude** [comment]
> This is my first PR to spellbook and would probably have botched something. Let me know if something is blocking

welcome!
a nice simple PR to start. we do have an automated source for erc20 token metadata, however, some coverage is missing at times. it can also be delayed a bit, so that's why we still have these manual files to compliment.

**Pull request merged**